tapes/submitters.tape

```diff
@@ -23,8 +30,8 @@ submitter sge :: action_flags
               :: TASK REALIZATION TASK_VARIABLES CONFIGURATION {
   action run {
     wrapper="ducttape_job.sh"
-    echo "#$ $resource_flags" >> $wrapper
-    echo "#$ $action_flags" >> $wrapper
+    echo "#$ resource_flags" >> $wrapper
```

Comment:
Why is `$` removed? Looks like the actual value of resource and action flags won't be passed anymore.

Reply:
typo